codes/c/chapter_backtracking/n_queens.c

```diff
@@ -0,0 +1,92 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdbool.h>
+#include <string.h>
```

Comment:
The header files are unified in `../utils/common.h`, so you only need
`#include "../utils/common.h"`